app/dashboard/models.py

```diff
@@ -2563,6 +2563,44 @@ def get_eth_sum(self, sum_type='collected', network='mainnet', bounties=None):
 
         return eth_sum
 
+    def get_all_tokens_sum(self, sum_type='collected', network='mainnet', bounties=None):
+        """Get the sum of collected or funded tokens based on the provided type.
+
+        Args:
+            sum_type (str): The sum to lookup.  Defaults to: collected.
+            network (str): The network to query results for.
+                Defaults to: mainnet.
+            bounties (dashboard.models.BountyQuerySet): Override the BountyQuerySet this function processes.
+                Defaults to: None.
+
+        Returns:
+            query: Grouped query by token_name and sum all token value
+        """
+        all_tokens_sum = None
+        if not bounties:
+            if sum_type == 'funded':
+                bounties = self.get_funded_bounties(network=network)
+            elif sum_type == 'collected':
+                bounties = self.get_fulfilled_bounties(network=network)
+            elif sum_type == 'org':
+                bounties = self.get_orgs_bounties(network=network)
+
+        if sum_type == 'funded':
+            bounties = bounties.has_funds()
```

Comment:
if `bounties` is empty and the `sum_type == 'funded'` this will fail

Reply:
bounties check's added